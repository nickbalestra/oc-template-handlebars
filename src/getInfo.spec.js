const getInfo = require('./getInfo');

describe('getInfo method', () => {
  describe('when invoking the method', () => {
    const info = getInfo();

    test('should return the correct version', () => {
      expect(info.version).toBe('1.0.0');
    });
    test('should return the correct template type', () => {
      expect(info.type).toBe('handlebars');
    });
    test('should return the list of dependencies', () => {
      const dependencies = { handlebars: '^4.0.6' };
      expect(info.dependencies).toMatchObject(dependencies);
    });
  });
});

