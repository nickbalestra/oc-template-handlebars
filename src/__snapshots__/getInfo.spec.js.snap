exports[`getInfo method when invoking the method should return the list of dependencies 1`] = `
Object {
  "handlebars": "^4.0.6",
}
`;

exports[`getInfo method when invoking the method should return the list of externals 1`] = `
Array [
  Object {
    "global": "Handlebars",
    "url": "https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.5/handlebars.runtime.min.js",
  },
]
`;
